src/main/java/com/demonwav/mcdev/platform/canary/CanaryModule.java

```diff
@@ -0,0 +1,55 @@
+package com.demonwav.mcdev.platform.canary;
+
+import com.demonwav.mcdev.buildsystem.BuildSystem;
+import com.demonwav.mcdev.platform.AbstractModule;
+import com.demonwav.mcdev.platform.PlatformType;
+import com.demonwav.mcdev.platform.canary.util.CanaryConstants;
+import com.intellij.openapi.module.Module;
```

Comment:
Check the style guide, `com.demonwav` imports should be separate. Use [this](https://github.com/DemonWav/MinecraftDev/blob/master/intellij-java-style.xml) to set the style properly in IntelliJ easilly.